src/day6.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <algorithm>
+
+
+std::vector<std::vector<std::string>> readPuzzleInputFromFile(const std::string& fileName) {
+    std::ifstream inFile(fileName);
+    std::string line;
+    std::vector<std::string> answersPerGroup;
+    std::vector<std::vector<std::string>> answers;
+    while(getline(inFile, line)) {
+        if(line.empty()) {
+            answers.push_back(answersPerGroup);
```

Comment:
Here, moving is probably more efficient, because `answersPerGroup` might contain a lot of data. Moving it would look like:

```cpp
answers.push_back(std::move(answersPerGroup));
```

This basically 'empties' `answersPerGroup` and *moves* the content into the `answers` vector.